debian/bitlbee-common.config

```diff
@@ -1,21 +0,0 @@
-#!/bin/sh
-
-set -e
-
-. /usr/share/debconf/confmodule
-[ -f /etc/default/bitlbee ] && . /etc/default/bitlbee
-
-db_title BitlBee
-
-if [ -n "$BITLBEE_PORT" ]; then
```

Comment:
Won't this break existing users that rely on $BITLBEE_PORT? Ideally this would automatically be migrated, but at the very least, this kind of change needs a warning in NEWS.Debian.